components/admin_console/admin_sidebar/admin_sidebar.jsx

```diff
@@ -262,6 +283,11 @@ export default class AdminSidebar extends React.Component {
                     }
                 }
 
+                if (this.state.sections != null) {
+                    if (this.state.sections.indexOf(`plugin_${p.id}`) === -1) {
```

Comment:
Is this check necessary here? It seems that the URL check is already covered [here](https://github.com/mattermost/mattermost-webapp/blob/20142963730312102b5084035ca24bdb613fad22/components/admin_console/admin_sidebar/admin_sidebar.jsx#L204) in the `renderRootMenu`

Reply:
Yes.. The root menu iterates over the static admin definitions. This is required when iterating the available pluings and render the matched ones.

Reply:
Gotcha!